utils.py

```diff
@@ -240,8 +240,7 @@ def get_random_int(upper_bound):
     """
     assert upper_bound >= 0 and isinstance(upper_bound, int)
 
-    generator = random.SystemRandom()
```

Comment:
Why remove this?

Reply:
If a generator is used I won't be able to set a seed so in order to set the seed and test I removed it.

Reply:
This is no longer an issue, right? Or if it is, you should mock it.

As a general rule, **please do not change code in core utility functions** just to make the tests on your PR pass. You risk breaking other parts of the application.